src/foam/core/crunch/ui/CapabilityWizardlet.js

```diff
@@ -95,6 +96,7 @@ foam.CLASS({
     {
       name: 'wao',
       hidden: true,
+      transient: true,
```

Comment:
This can not be transient, it is expected that the server will send this to the client